src/SFA.DAS.ProviderApprenticeshipsService.Application/Queries/GetTrainingProgrammes/GetTrainingProgrammesQueryHandler.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using MediatR;
+using SFA.DAS.ProviderApprenticeshipsService.Domain.Interfaces;
+using SFA.DAS.ProviderApprenticeshipsService.Domain.Models.ApprenticeshipCourse;
+
+namespace SFA.DAS.ProviderApprenticeshipsService.Application.Queries.GetTrainingProgrammes
+{
+    public sealed class GetTrainingProgrammesQueryHandler : IAsyncRequestHandler<GetTrainingProgrammesQueryRequest, GetTrainingProgrammesQueryResponse>
+    {
+        private readonly IApprenticeshipInfoServiceWrapper _apprenticeshipInfoServiceWrapper;
+
+        public GetTrainingProgrammesQueryHandler(IApprenticeshipInfoServiceWrapper apprenticeshipInfoServiceWrapper)
+        {
+            _apprenticeshipInfoServiceWrapper = apprenticeshipInfoServiceWrapper;
+        }
+
+        public async Task<GetTrainingProgrammesQueryResponse> Handle(GetTrainingProgrammesQueryRequest message)
+        {
+            var standardsTask = _apprenticeshipInfoServiceWrapper.GetStandardsAsync();
+            var frameworksTask = message.IncludeFrameworks
```

Comment:
Same suggestion here as made in the employer commitments PR..
https://github.com/SkillsFundingAgency/das-employercommitments/pull/165